app/index.html

```diff
@@ -1,291 +1,401 @@
+            </li>
+            <li class="category-item">
+              <div class="category-img">
+                <img
+                  class="cat-img"
+                  src="./assets/pngs/icons/html.png"
+                  alt=""
+                />
+              </div>
+              <span class="cat-name">HTML</span>
+            </li>
+            <li class="category-item">
+              <div class="category-img">
+                <img
+                  class="cat-img"
+                  src="./assets/pngs/icons/css.jpeg"
+                  alt=""
+                />
+              </div>
+              <span class="cat-name">CSS</span>
+            </li>
+          </ul>
+          <span class="section-title">
+            <div class="title-text">
+              <p>Frameworks</p>
             </div>
-            <span class="cat-name">HTML</span>
-          </li>
-          <li class="category-item">
-            <div class="category-img">
-              <img class="cat-img" src="./assets/pngs/icons/css.jpeg" alt="">
+            <span class="dropdown-icon"
+              ><img src="./assets/pngs/icons/dropBtn.png" alt=""
+            /></span>
+          </span>
+          <div class="divide"></div>
+          <span class="section-title">
+            <div class="title-text">
+              <p>Skills</p>
             </div>
-            <span class="cat-name">CSS</span>
-          </li>
-        </ul>
-        <span class="section-title">
-          <div class="title-text">
-            <p>Frameworks</p>
+            <span class="dropdown-icon"
+              ><img src="./assets/pngs/icons/dropBtn.png" alt=""
+            /></span>
+          </span>
+          <div class="divide"></div>
+        </div>
+      </section>
+      <!-- about section ends here -->
+      <!-- contact section starts here -->
+      <section class="contact-section">
+        <div class="contact-container">
+          <div class="contact-title">
+            <h2>Contact Me</h2>
           </div>
-          <span class="dropdown-icon"><img src="./assets/pngs/icons/dropBtn.png" alt=""></span>
-        </span>
-        <div class="divide"></div>
-        <span class="section-title">
-          <div class="title-text">
-            <p>Skills</p>
+          <div class="contact-body">
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding i'd
+              love to help with it.
+            </p>
           </div>
-          <span class="dropdown-icon"><img src="./assets/pngs/icons/dropBtn.png" alt=""></span>
-        </span>
-        <div class="divide"></div>
-      </div>
-    </section>
-    <!-- about section ends here -->
-  </div>
-</body>
-
-</html>
\ No newline at end of file
+          <div class="form-container">
+            <form action="#" method="POST">
+              <div class="input-container">
+                <input
```

Comment:
Make sure that the `POST` method of your form is working properly, the user should be able to send a message using  the `action` and `method` attribute  integrated with [formspree service](https://formspree.io/html).

* `action` value should be similar to `https://formspree.io/f/{form_id}`
* `method` value equals `POST`Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).

Please, follow the link above and make sure the response is `successful`